### DISCUSSION TITLE
Independent Workspaces per monitor

### DISCUSSION THREAD

**Discussion by p-sims** [elementary/os#718] [Ideas]
MacOS has had this for a decade.
GNOME has been [discussing this for a while](https://gitlab.gnome.org/GNOME/mutter/-/issues/37) but is yet to implement.
System76 are [implementing on Pop!](https://github.com/pop-os/cosmic-epoch/issues/53) when Cosmic is released. 

I understand that it's difficault to implement under X11 but as Elementary is shifting to Wayland that make it easier....

I'd like to suggest Elementary adds support for multiple independent workspaces per Display in a future release. I couldn't see a simular discussion on this, so I wanted to start one.

**Comment by ryonakano** [comment]
Closing as duplicate of https://github.com/elementary/gala/issues/26
